packages/react-components/theme-designer/src/components/Content/Content.stories.tsx

```diff
@@ -1,3 +1,10 @@
 import { Content } from './Content';
+import { getBrandTokensFromPalette } from '../../utils/getBrandTokensFromPalette';
+import { createLightTheme, createDarkTheme } from '@fluentui/react-components';
 export default { component: Content };
-export const Default = {};
+
+const brand = getBrandTokensFromPalette('#006BC7', {});
```

Comment:
2nd param shouldn't be required
```suggestion
const brand = getBrandTokensFromPalette('#006BC7');
```